packages/high-roller-bot/.gitignore

```diff
@@ -0,0 +1,2 @@
+src/store.json
+src/settings.json
```

Comment:
Why is this here

Reply:
@snario Why should this not be in .gitignore?
Do we want to commit `store.json` and `settings.json`? I am assuming we don't...